cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -66,13 +66,23 @@ set(configure_command
 # Pass C flags through
 set(configure_opts ${configure_opts} ${CMAKE_C_FLAGS})
 
+# Pass BUILD_SHARED_LIBS (what about RPATH?)
+set(shared_libs "@BUILD_SHARED_LIBS@")
+if (shared_libs)
```

Comment:
With ```if(BUILD_SHARED_LIBS)```, this does not work.

```
hunter/examples/OpenSSL$ cmake -H. -B_build -DBUILD_SHARED_LIBS=1 -DHUNTER_STATUS_DEBUG=1 |& tee log.txt
```

[log.txt](https://github.com/ruslo/hunter/files/850537/log.txt)

You can see that it picks "static libs"